src/algorithms/utils/TimeSeries.test.ts

```diff
@@ -1,6 +1,106 @@
 import { uniformDatesBetween, makeTimeSeries, updateTimeSeries } from './TimeSeries'
 import { interpolateTimeSeries } from '../run'
 
+import * as fs from 'fs'
+const { utils } = require('jest-snapshot')
+
+expect.extend({
+  toBeCloseToArraySnapshot(received: number[]) {
+    const { testPath, currentTestName, snapshotState } = this
```

Comment:
Jest matchers are invoked with a context, which contains a mutable 'snapshotState' object and the test name. We need those in order to continue.